infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py

```diff
@@ -58,16 +60,6 @@ def main():
   # The default return code when an error occurs.
   error_code = 1
   if dry_run:
```

Comment:
if the build fails in the dry run, will we still attempt to run the fuzzers?

Reply:
Yes we will because there is no notification between actions with out a failure error code. The run_fuzzers should exit gracefully because It just means there are no fuzz targets where it expects to find them.

Reply:
Ack